From 8c14393f11081f2c13bbbf495e7a4e3ba192d64c Mon Sep 17 00:00:00 2001
From: Chenxi Tong <fchenxi@126.com>
Date: Wed, 9 Aug 2017 12:00:53 +0800
Subject: [PATCH 427/438] HBASE-18125 shell disregards spaces at the end of a
 split key in a split file

Signed-off-by: fchenxi <fchenxi@126.com>
Signed-off-by: Michael Stack <stack@apache.org>
(cherry picked from commit a17ed0356f12c6f7a682557d92cc401b7a4297f1)

Change-Id: I117f4d6e3badb0ab75f9ba2deea298a03bd41359
Author: Chenxi Tong
Reason: Bug
Ref: CDH-61569
---
 hbase-shell/src/main/ruby/hbase/admin.rb |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hbase-shell/src/main/ruby/hbase/admin.rb b/hbase-shell/src/main/ruby/hbase/admin.rb
index 3e4ea11..140ab68 100644
--- a/hbase-shell/src/main/ruby/hbase/admin.rb
+++ b/hbase-shell/src/main/ruby/hbase/admin.rb
@@ -304,7 +304,7 @@ module Hbase
           end
           arg[SPLITS] = []
           File.foreach(splits_file) do |line|
-            arg[SPLITS].push(line.strip())
+            arg[SPLITS].push(line.chomp)
           end
           htd.setValue(SPLITS_FILE, arg[SPLITS_FILE])
         end
-- 
1.7.9.5

